features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/index.jsx

```diff
@@ -753,14 +753,16 @@ class Details extends Component {
                                             to={pathPrefix + 'business info'}
                                             Icon={<BusinessIcon />}
                                         />
-                                        <LeftMenuItem
-                                            text={intl.formatMessage({
-                                                id: 'Apis.Details.index.subscriptions',
-                                                defaultMessage: 'subscriptions',
-                                            })}
-                                            to={pathPrefix + 'subscriptions'}
-                                            Icon={<SubscriptionsIcon />}
-                                        />
+                                        {!api.advertiseInfo.advertised && (
```

Comment:
Let's check whether `advertiseInfo` is exist 

```suggestion
                                        {api.advertiseInfo && !api.advertiseInfo.advertised && (
```

Reply:
Tested locally, Without this test page will break when accessing any API product details page 
![image](https://user-images.githubusercontent.com/3313885/114400662-d0425900-9bbf-11eb-86cc-d6ce59ad1b1d.png)

Reply:
Fixed. 
![apiProd](https://user-images.githubusercontent.com/69344030/114771092-cd935f80-9d89-11eb-819c-3b2004537472.png)